animation_nodes/math/rotation_conversion.pxd

```diff
@@ -6,4 +6,9 @@ from . vector cimport Vector3
 
 cdef matrixToEuler(Euler3* target, Matrix3_or_Matrix4* m)
 cdef void normalizedAxisAngleToMatrix(Matrix3_or_Matrix4* m, Vector3* axis, float angle)
-cdef void normalizedAxisCosAngleToMatrix(Matrix3_or_Matrix4* m, Vector3* axis, float cosAngle)
\ No newline at end of file
+cdef void normalizedAxisCosAngleToMatrix(Matrix3_or_Matrix4* m, Vector3* axis, float cosAngle)
+cdef euler3ToQuaternion(Quaternion* q, Euler3 *e)
+cdef quaternionToMatrix4(Matrix4 *m, Quaternion *q)
+cdef quaternionToEuler3(Euler3 *e, Quaternion *q)
```

Comment:
Add `void` here. Put the `*` next to the parameter name, for instance `Quaternion* q` -> `Quaternion *q`.